### PR TITLE
Init

### PR DESCRIPTION
**Pull request by doino-gretchenliev**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - DISTRO=centos-6-x64
+  - DISTRO=centos-7-x64
+  - DISTRO=debian-7-x64
+  - DISTRO=debian-8-x64
+  - DISTRO=ubuntu-server-1204-x64
+  - DISTRO=ubuntu-server-1404-x64
+script:
+- make test-$TEST_SUITE
+before_install:
+- openssl aes-256-cbc -K $encrypted_c1be2d1aa63f_key -iv $encrypted_c1be2d1aa63f_iv
```
Comment (nikolavp) [inline .travis.yml]: Can you please say something more on how this works exactly?

Comment (doino-gretchenliev) [inline .travis.yml]: Nope, this file is not needed anymore. We're using jenkins for matrix builds

File: lib/puppet/type/graphdb_link.rb [hunk 2]
```diff
@@ -0,0 +1,84 @@
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', '..', '..'))
```
Comment (nikolavp) [inline lib/puppet/type/graphdb_link.rb]: Same comment as above for the graphdb_link.

**Comment by nikolavp** [comment]
There was a scripts that I have written for the previous module
generate-repository-parameters.py
Is there a reason why it is not included. We should somehow be able to update the turtle configuration which btw are currently out of date

**Pull request merged**